src/components/modal/InputLlm/index.tsx

```diff
@@ -19,8 +24,20 @@ const InputLlm: React.FC = () => {
 
   const dispatch = useDispatch();
 
+  async function gptFunction() {
```

Comment:
consider separating the logics from component if the computing logic becomes too complicated

Reply:
Nice advice! But i have no idea how to separate.
I put them together because I need them to async/await the previous function.

Reply:
Let's wrap the three functions `1`, `2`, `3` in one async function `A` and then expose the function `A`